qcweb/templates/layout.html

```diff
@@ -33,9 +33,9 @@
               <a class="nav-item nav-link" href="/plot">Plot</a>
             </div>
             <!-- Navbar Right Side -->
-            <div class="navbar-nav">
-                <a class="nav-item nav-link" href="/search">Search</a>
-            </div>
+            <!--div class="navbar-nav"-->
```

Comment:
I think we should just delete lines 35-38. It would be easy to add the element back if there was any need for it.